RecoTracker/IterativeTracking/python/DisplacedRegionalStep_cff.py

```diff
@@ -453,6 +453,7 @@
 from RecoTracker.FinalTrackSelectors.trackSelectionTf_cfi import *
 from RecoTracker.FinalTrackSelectors.trackSelectionTf_CKF_cfi import *
 trackdnn.toReplaceWith(displacedRegionalStep, trackTfClassifier.clone(
+     mva         = dict(tfDnnLabel  = 'trackSelectionTfDisplacedRegional'),
      src         = 'displacedRegionalStepTracks',
      qualityCuts = qualityCutDictionary.DisplacedRegionalStep.value()
```

Comment:
are the cuts (IIRC, taken from tobtec) in the `qualityCutDictionary` accidentally OK with this new training?

Reply:
Yes, the cuts have not been retuned, so the copied values seem to be coincidentally OK.

I'm not actually sure how the specific cut values should be chosen, but I guess the gain from any optimization would be marginal.